pkgs/by-name/sw/switch-to-configuration-ng/src/src/main.rs

```diff
@@ -1037,12 +1053,13 @@ fn do_system_switch(action: Action) -> anyhow::Result<()> {
         die();
     };
 
+    log::debug!("Acquiring lock on file /run/nixos/switch-to-configuration.lock");
```

Comment:
Might be worth it to factor `/run/nixos/switch-to-configuration.lock` out to a constant.